spinalcordtoolbox/scripts/sct_analyze_lesion.py

```diff
@@ -333,6 +335,32 @@ def _measure_axial_damage_ratio(self, im_data, p_lst, idx):
         printv('  Maximum axial damage ratio: ' + str(np.round(maximum_axial_damage_ratio, 2)),
                self.verbose, type='info')
 
+    def _compute_issue_bridge_ratio(self, idx):
+        """
+        :param idx: int, index of the lesion
+        Compute tissue bridge ratios:
+        - ventral_bridge_ratio: ventral midsagittal tissue bridge width divided by total midsagittal tissue bridge width
+        - dorsal_bridge_ratio: dorsal midsagittal tissue bridge width divided by total midsagittal tissue bridge width
+        """
+        # Get the dorsal and ventral tissue bridges
+        dorsal_bridge_width = float(self.measure_pd.loc[idx, 'interpolated_dorsal_bridge_width [mm]'])
+        ventral_bridge_width = float(self.measure_pd.loc[idx, 'interpolated_ventral_bridge_width [mm]'])
+
+        # Compute the total tissue bridge width
+        total_bridge_width = dorsal_bridge_width + ventral_bridge_width
+
+        # Compute the dorsal and ventral bridge ratios
+        dorsal_bridge_ratio = dorsal_bridge_width / total_bridge_width * 100 if total_bridge_width > 0 else 0
+        ventral_bridge_ratio = ventral_bridge_width / total_bridge_width * 100 if total_bridge_width > 0 else 0
+
+        # Save the ratios to the measure DataFrame
+        self.measure_pd.loc[idx, 'dorsal_bridge_ratio [%]'] = dorsal_bridge_ratio
+        self.measure_pd.loc[idx, 'ventral_bridge_ratio [%]'] = ventral_bridge_ratio
```

Comment:
This should be something we can test fairly easily, right? Since we're already checking both bridge widths in the "compute_expected_measurements" function, the calculation of the "expected" ratio seems very straightforward too. :D

Reply:
That's an excellent point! I actually started writing a test for these new ratio metrics, but then I realized that we are actually not computing (and testing) `interpolated_dorsal_bridge_width` and `interpolated_ventral_bridge_width` in our test suite. These interpolated bridges are used to compute `dorsal_bridge_ratio` and `ventral_bridge_ratio`:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/0f97779b53b3f81d6698b7382920f9914fba290c/spinalcordtoolbox/scripts/sct_analyze_lesion.py#L345-L347 

In other words, if we want to test `dorsal_bridge_ratio` and `ventral_bridge_ratio`, we need to add computation of `interpolated_dorsal_bridge_width` and `interpolated_ventral_bridge_width`.

Reply:
> ```python
>             # We don't actually store the total interpolated bridge width in the final output, so delete it
>             del tissue_bridges['interpolated_total_bridge_width [mm]']
> ```

That's a good point, actually! Total interpolated bridge width is also useful, so why not store it? I added it to the main script in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4933/commits/69dc10d72000c1be4ea12a87371814ad5245b82f.

Reply:
> It's okay! I spent a little time implementing the test, and it seems to work OK?

This is awesome, @joshuacwnewton! Thank you a lot! I took the liberty of including your code in my recent commits.

Reply:
> The values are the same, but one value has slightly more precision than the other, so it is greater than expected. I'm not quite sure the best fix for this? Maybe rounding both values to the same number of decimal points before comparing? 🤔

Good point! I tried rounding to 10 decimal places, and it works, so I implemented it in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4933/commits/f28111b5a40ac1ac8335815fad324dcb8ff11f02